optimade/server/mappers/structures.py

```diff
@@ -0,0 +1,51 @@
+from configparser import ConfigParser
+from pathlib import Path
+
+
+config = ConfigParser()
+config.read(Path(__file__).resolve().parent.parent.joinpath("config.ini"))
+PROVIDER = config["DEFAULT"].get("PROVIDER")
+PROVIDER_FIELDS = {field for field, _ in config["STRUCTURE"].items() if _ == ""}
+
+
+class StructureMapper:
```

Comment:
Yeah - I have done a lot of tweaks and stuff.
Most of it started on the spec, but then sort of propagated along, since I wanted the "test" server to actually run... Sorry :sweat_smile: 

I think the aliases should come from config files (or similar) as well - it makes most sense.
In this case, I just think that `StructureMapper` has nothing to do with the spec, but rather the implementation, and as such should never be in /models/*

Reply:
Definitely not a complaint, feels like we're making good progress! Just feel sorry for anyone else who reviews this!

Reply:
> (...) Just feel sorry for anyone else who reviews this!

Hahaha :rofl: yeah - poor sods ...

In any case, I suggest we have a quick Skype call to lay out what should be done to finish the PR and then let's send it to review?